pkg/quota/admission/clusterresourcequota/accessor.go

```diff
@@ -147,10 +148,14 @@ func (e *clusterQuotaAccessor) waitForReadyClusterQuotaNames(namespaceName strin
 	err := utilwait.PollImmediate(100*time.Millisecond, 8*time.Second, func() (done bool, err error) {
 		var namespaceSelectionFields clusterquotamapping.SelectionFields
 		clusterQuotaNames, namespaceSelectionFields = e.clusterQuotaMapper.GetClusterQuotasFor(namespaceName)
-		namespace, err := e.namespaceLister.Get(namespaceName)
+		obj, ok, err := e.namespaceLister.Get(&kapi.Namespace{ObjectMeta: kapi.ObjectMeta{Name: namespaceName}})
```

Comment:
> You can't call a Get() with a string. The semantics are wrong?

The kube code is wrong and you're handling the `!ok` incorrectly.  As I recall `!ok` is actually a "NotFoundError"